.github/workflows/ci.yml

```diff
@@ -133,7 +139,17 @@ jobs:
     - name: Build
       shell: bash
       run: cmake --build build --config ${{ matrix.type.name == 'Debug' && 'Debug' || 'Release' }} --target install
-      
+
+    - name: Build Android example
+      if: matrix.platform.name == 'Android'
+      shell: bash
+      run: |
+        cd examples/android
```

Comment:
Is it possible to do this without changing directories? If this is required, let's be sure to run `cd -` at the end of this job so that the current directory isn't affected for later jobs.

Reply:
Following steps are not impacted by `cd`.
> Each run keyword represents a new process and shell in the runner environment.

See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepsrun

It is possible to replace `cd` by `working-directory:`.

Reply:
It can be done without changing directory, but I'll change it to `working-directory` so that the path doesn't need to be repeated on each line.

Reply:
To clarify, you wouldn't have to repeat `cd` as long as you are in the same `run` step. You can have several commands in a `run` step as you do, which are executed in the same shell. I was answering the ChrisThrasher who was worried it would impact the rest of the workflow. But `working-directory` is an alternative and looks nice.